ui/tests/helpers/login.ts

```diff
@@ -0,0 +1,3 @@
+export default function login(token?: string) {
+  window.localStorage.waypointAuthToken = token || 'default-test-token-value';
```

Comment:
Nice, do you need to clear this between tests?

Reply:
Yes, thanks, will todo.

Reply:
mentioned this, but thought I'd add the link - you may want to look into https://github.com/kaliber5/ember-window-mock